src/components/main-layout/ContentPanelTimer.tsx

```diff
@@ -0,0 +1,111 @@
+    FullyVisible = 3,
+}
+
+const ContentPanelTimer: React.FC<ContentPanelProps> = ({
+    pageContent,
+    currentModule,
+}) => {
+    const [renderState, setRenderState] = useState(RenderState.NewContent);
+    const previousContentRef = useRef(pageContent);
+
+    useEffect(() => {
+        // There are pages that have the same written content, so flashing the
+        // panel is more distracting than helpful. However, since this Ref is defaulted
+        // to the first page content, this check will always be true on the first page, and
+        // will keep the content from being rendered, so it needs to be excluded
+        if (!contentJustChanged && page !== 1) {
+            previousContentRef.current = pageContent;
+            return;
+        }
+        const count: number = 1;
+        const FADE_TIME = 100;
+        let timer: NodeJS.Timeout;
+        const updateRenderState = (count: number) => {
+            previousContentRef.current = pageContent;
+            setRenderState(count);
+            count = count + 1;
+
+            if (count <= RenderState.FullyVisible) {
+                timer = setTimeout(() => updateRenderState(count), FADE_TIME);
+            } else {
+                return () => clearTimeout(timer);
+            }
+        };
+        timer = setTimeout(() => updateRenderState(count), FADE_TIME);
+        return () => clearTimeout(timer);
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, [pageContent.content]);
+
+    const contentJustChanged = !isEqual(
+        previousContentRef.current.content,
+        pageContent.content
+    );
+    // This allows us to fade out the previous content when it's been change.
+    // it will momentarily render the previous content before fading out
+    const contentToUse = contentJustChanged
+        ? previousContentRef.current
+        : pageContent;
+
+    const {
+        content,
+        title,
+        callToAction,
+        backButton,
+        nextButton,
+        nextButtonText,
+        moreInfo,
+        actionButton,
+        section,
+        layout,
+    } = contentToUse;
+
+    const { page } = useContext(SimulariumContext);
+    const pageNumber = contentJustChanged ? page - 1 : page;
+    const containerClassNames = classNames([
+        styles.contentPanelWrapper,
+        {
+            [styles.fadeOut]: contentJustChanged,
+        },
+        { [styles.loaded]: renderState >= RenderState.StartFade },
+    ]);
+
+    return (
+        <ContentPanel
+            pageNumber={pageNumber}
+            title={title}
+            layout={layout}
+            content={content}
+            callToAction={callToAction}
+            backButton={backButton}
+            nextButton={nextButton}
+            nextButtonText={nextButtonText}
+            moreInfo={moreInfo}
+            actionButton={actionButton}
+            section={section}
+            currentModule={currentModule}
+            containerClass={containerClassNames}
```

Comment:
```suggestion
            {...contentToUse}
            pageNumber={pageNumber}
            containerClass={containerClassNames}
```

^ check if this works